bson/bsontype/bsontype.go

```diff
@@ -47,6 +47,7 @@ const (
 	BinaryMD5         byte = 0x05
 	BinaryEncrypted   byte = 0x06
 	BinaryColumn      byte = 0x07
+	BinarySenstive    byte = 0x08
```

Comment:
These types are deprecated, BinarySensitive needs to be referenced in the bson package [here](https://github.com/mongodb/mongo-go-driver/blob/master/bson/types.go#L39) .

Reply:
Fixed